docs/services/motion/_index.md

```diff
@@ -437,8 +437,6 @@ Translation in obstacles is not supported by the [navigation service](/services/
 - `movement_sensor_name` [(ResourceName)](https://python.viam.dev/autoapi/viam/gen/common/v1/common_pb2/index.html#viam.gen.common.v1.common_pb2.ResourceName): The `ResourceName` of the [movement sensor](/components/movement-sensor/) that you want to use to check the robot's location.
 - `obstacles` [(Optional[Sequence[GeoObstacle]])](https://python.viam.dev/autoapi/viam/gen/common/v1/common_pb2/index.html#viam.gen.common.v1.common_pb2.GeoObstacle): Obstacles to consider when planning the motion of the component, with each represented as a `GeoObstacle`. <ul><li> Default: `None` </li></ul>
 - `heading` [(Optional[float])](https://docs.python.org/library/typing.html#typing.Optional): The compass heading, in degrees, that the robot's movement sensor should report at the `destination` point. <ul><li> Range: `[0-360)` </li><li>Default: `None`</li></ul>
-- `linear_meters_per_sec` [(Optional[float])](https://docs.python.org/library/typing.html#typing.Optional): The linear velocity in meters per second to target when moving the component. <ul><li> Default: `0.3` </li></ul>
-- `angular_deg_per_sec` [(Optional[float])](https://docs.python.org/library/typing.html#typing.Optional): The [angular velocity](https://en.wikipedia.org/wiki/Angular_velocity) in degrees per second to target when turning the component. <ul><li> Default: `60` </li></ul>
```

Comment:
AFAIK These are in error on the python SDK docs, they are now actually part of motion_configuration: same thing in Go. See other pr linked in description for more info